rootless.md

```diff
@@ -0,0 +1,31 @@
+# Shortcomings of Rootless Podman
+
+* Lacking “How To” documentation or documentation in general
+* If /etc/subuid and /etc/subgid not setup for a user, then podman commands
+can easily fail
+  * This can be a big issue on machines using Network Based Password information (FreeIPA, Active Directory, LDAP)
+  * We are working to get support for NSSWITCH on the /etc/subuid and /etc/subgid files.
+* Podman can not create containers that bind to ports < 1024.
+  * The kernel does not allow processes without CAP_NET_BIND_SERVICE to bind to low ports.
+* No cgroup Support (hopefully fixed when cgroups V2 happens).
+  * Cgroups V1 does ont support cgroup delegation.
+* Can not share container images with CRI-O or other users
+* Difficult to use additional stores for sharing content
+* Does not work on NFS homedirs
+* Does not work with homedirs mounted with noexec/nodev
+* Can not use overlayfs driver, but does support fuse-overlayfs
+  * Does not perform as well as root podman with Kernel based Overlay
+* Only other supported driver is VFS.
+* No KATA Container support ???
+* Difficult to use multiple different User Namespaces
```

Comment:
this is going to be fixed with the refactoring I am working on, I'd probably just drop this line for now

Reply:
Removing.